3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_structures_geojson.psql

```diff
@@ -0,0 +1,72 @@
+-- Deploy reveal_irs_zambia_2020:zambia_irs_structures_geojson to pg
+-- requires: reveal_irs_zambia_2019:zambia_irs_structures
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+-- We are creating a materialized view that exposes data from zambia_irs_structures
+-- as a materialized view withput the geometry column.  This is being done because
+-- in production zambia_irs_structures is not being refreshed concurrently and so we
+-- needs its data available for reporting even during its refreshing.
+CREATE MATERIALIZED VIEW IF NOT EXISTS zambia_irs_structures_data
```

Comment:
Naming - we should avoid using "materialized_view", "data", "table", etc. in our names - they're mostly confusing.  zambia_irs_structures_report is a convention I've been using, for stuff that is used for reporting, here you could do "zambia_irs_structures_report_no_geojson" and then have the view below be "zambia_irs_structures_report".

Also, that way we'd have an easy naming-based sanity check on our refresh logic - non-reports get refreshed non-concurrently, reports get refreshed concurrently.

If you don't like _report, something like "_nonblocking" or "_live" could work but I feel like report has more semantic value.

Reply:
Will get a PR your way changing this